packages/editor/src/lib/license/LicenseManager.test.ts

```diff
@@ -417,7 +417,7 @@ function importPrivateKey(pemContents: string) {
 	// base64 decode the string to get the binary data
 	const binaryDerString = atob(pemContents)
 	// convert from a binary string to an ArrayBuffer
-	const binaryDer = str2ab(binaryDerString) as Uint8Array
```

Comment:
how did you find this tweak? were you looking for extra casting?

Reply:
with typescript getting bumped this became an error